magenta/musescore/magenta.qml

```diff
@@ -0,0 +1,109 @@
+import QtQuick 2.0
+import MuseScore 1.0
+
+MuseScore {
+      menuPath: "Plugins.Magenta"
+
+      function createNoteSequence() {
+        var noteSequence = {
+          timeSignatures: [],
+          keySignatures: [],
+          tempos: [],
+          notes: [],
+          totalTime: curScore.duration,
+          sourceType: 'SCORE',
```

Comment:
I'll modify the source field in the proto to include a string name in addition to the source enum.